### DISCUSSION TITLE
openrgb + themeswitcher.sh

### DISCUSSION THREAD

**Discussion by T-Crypt** [prasanthrangan/hyprdots#135] [Ideas]
Going to start working on the openrgb + themeswitcher.sh integration unless you have already started this.

Looks like you can create openrgb profiles -- if I can get the profiles created for each dynamic theme, should be able to call the profiles for each theme in the themeswitcher.sh script. 

May be even more simple is just setting the static mode and color and adding a 'openrgb-theme wiki' section on modifying the script to change the mode. (static, breathing, 'Color Cycle')

openrgb --mode static --color 3d85c6 (works to set my devices to match frosted-glass)

**Comment by T-Crypt** [comment]
I use corsair devices for my setup so this may be a little more in depth to grab CKB-Next daemon and OpenRGB to integrate but Ill keep working on this.

**Comment by T-Crypt** [comment]
This sets the keyboard as active -- changes the color for any connected devices up to 9 devices as the CKB-Next allows up to 9. Then just sets the openrgb color 

Can incorporate this into the script but will just need to modify the color code for each theme!

![Screenshot 2023-08-30 102736](https://github.com/prasanthrangan/hyprdots/assets/132922589/d8a25f35-fe87-4a8f-a47b-e3f949bf9ead)

also -- need to add 'openrgb' to custom_hypr.lst

**Comment by prasanthrangan** [comment]
Hi, rgb settings can be very system specific, so the user has to make profiles for his system/devices that support it, I'm not sure if a generic profile can be applied use to any device... So better keep openrgb as optional and not part of main install, to keep it as minimal as possible.
